src/main/java/blackjack/domain/gambler/Gambler.java

```diff
@@ -0,0 +1,74 @@
+package blackjack.domain.gambler;
+
+import blackjack.domain.WinningDiscriminator;
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public abstract class Gambler {
+    private static final int MIN_ACE_VALUE = 1;
+    private final List<Card> cards;
+    private final Name name;
+
+    public Gambler(final Name name) {
+        this.name = name;
+        this.cards = new ArrayList<>();
+    }
+
+    public void addCard(final Card card) {
+        cards.add(card);
+    }
+
+    public boolean isNameEquals(final Name name) {
+        return Objects.equals(name, this.name);
+    }
+
+    public boolean isScoreBelow(final int criteria) {
+        return calculateScore() <= criteria;
+    }
+
+    public int calculateScore() {
+        int sum = calculateSum();
+        long aceCount = calculateAceCount();
+        if (aceCount == 0) {
+            return sum;
+        }
+        return adjustSumByAce(sum, (int) aceCount);
+    }
+
+    private int calculateSum() {
+        return cards.stream()
+                .mapToInt(Card::getTypeValue)
+                .sum();
+    }
+
+    private long calculateAceCount() {
+        return cards.stream()
+                .filter(Card::isAce)
+                .count();
+    }
+
+    protected int adjustSumByAce(int sum, int aceCount) {
+        int differenceBetweenMaxAndMinValue = CardType.ACE.getValue() - MIN_ACE_VALUE;
+        sum -= differenceBetweenMaxAndMinValue * (aceCount - 1);
+
+        if (sum > WinningDiscriminator.BLACK_JACK) {
+            sum -= differenceBetweenMaxAndMinValue;
+        }
+        return sum;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public List<Card> getCards() {
+        return Collections.unmodifiableList(cards);
+    }
+
+    public abstract List<Card> getInitialCards();
```

Comment:
문득 코드를 보다가 든 생각인데, Name으로 이미 Dealer와 Player를 구분할 수 있으니 Gambler에 getInitialCards()를 아래처럼 구현해볼 수도 있겠군요. 

```suggestion
    public List<Card> getInitialCards() {
        if (name.isDealer()) {
            List<Card> cards = getCards();
            Card firstCard = cards.getFirst();
            return List.of(firstCard);
        }
        return getCards();
    }
```

그러면 Gambler를 추상 클래스로 만들 필요가 없을 것 같습니다! step2를 진행해보면서 같이 고려해보세요~